ONSdigital/perkin
# Updated Restolino and exposed self-documentation on the default route.

## Changes

Updated Restolino to 0.1.14, which provides API self-documentation in Json on the default (/) route.

## How to test

Build Perkin and go to localhost:8080 - you should see the methods and request/response messages.

## Who can test

Anyone but @davidcarboni
